config/core/test-configs.yaml

```diff
@@ -179,6 +179,16 @@ test_plans:
       flashing_rootfs: http://storage.chromeos.kernelci.org/images/rootfs/chromeos/20220412.0/bullseye-chromeos-flash/amd64/full.rootfs.tar.xz
       flashing_ramdisk: http://storage.chromeos.kernelci.org/images/rootfs/chromeos/20220412.0/bullseye-chromeos-flash/amd64/initrd.cpio.gz
 
+  chromeos-install-modules:
+    pattern: 'chromeos/chromeos-install-modules-template.jinja2'
+    params:
+      job_timeout: '60'
+      modules_url: http://images.collabora.com/lava/chromeos/modules-4.14.243.tar.gz
+      flashing_kernel: http://storage.chromeos.kernelci.org/images/kernel/v5.17.2-x86_64+x86_chromebook/kernel/bzImage
+      flashing_modules: http://storage.chromeos.kernelci.org/images/kernel/v5.17.2-x86_64+x86_chromebook/modules.tar.xz
+      flashing_rootfs: http://storage.chromeos.kernelci.org/images/rootfs/chromeos/20220412.0/bullseye-chromeos-flash/amd64/full.rootfs.tar.xz
```

Comment:
The URL should have been:
http://storage.chromeos.kernelci.org/images/rootfs/chromeos/bullseye-chromeos-flash/20220412.0/amd64/full.rootfs.tar.xz
I guess that's something to fix in a follow-up PR with an image built and uploaded to the right location.

Reply:
Exactly.
The URL is dependent on a location where the Jenkins job pushes the image. It may be worth filing an issue to fix the Jenkins job in the first place.